discovery-provider/compose/env/shellEnv1.sh

```diff
@@ -3,3 +3,4 @@ export audius_server_port="5000"
 export audius_db_port="5432"
 export audius_ipfs_external_port="5200"
 export audius_redis_port="5380"
+export audius_es_port="9200"
```

Comment:
The convention we'd had is to put all the ports per service in the thousand's range 
ie. discovery is all in 5000, identity in 7000, content in 4000
but I think this is fine